client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -73,11 +73,9 @@
             if (card.type === 'search' && card.search && card.search.filter.query) {
                 params = card.search.filter.query;
                 if (card.query) {
-                    if (card.search.filter.query.q) {
-                        params.q = '(' + card.query + ') ' + card.search.filter.query.q;
-                    } else {
-                        params.q = '(' + card.query + ') ';
-                    }
```

Comment:
You have to keep card.search.filter.query.q, because it is from saved search groups type.